rhino-tools/src/main/java/org/mozilla/javascript/tools/debugger/Dim.java

```diff
@@ -542,7 +543,7 @@ private Object getObjectPropertyImpl(Context cx, Object object, Object id) {
             String name = (String) id;
             if (name.equals("this")) {
                 result = scriptable;
-            } else if (name.equals("__proto__")) {
+            } else if (name.equals(NativeObject.PROTO_PROPERTY)) {
                 result = scriptable.getPrototype();
             } else if (name.equals("__parent__")) {
```

Comment:
I'm all for using proper constants -- if we're going to do this for "__proto__", shouldn't we also do it for "__parent__"?